app/src/main/res/values/strings.xml

```diff
@@ -116,4 +116,63 @@
   <string name="empty_forecast_list_server_error">No weather information available. The server is not returning valid data. Please check for an updated version of Sunshine.</string>
   <string name="empty_forecast_list_invalid_location">No weather information available. The location in settings is not recognized by the weather server.</string>
 
+  <!-- A11y -->
+  <string name="a11y_forecast">Forecast: <xliff:g id="condition">%1$s</xliff:g></string>
+  <string name="a11y_forecast_icon">Forecast icon: <xliff:g id="condition">%1$s</xliff:g></string>
+  <string name="a11y_high_temp">High: <xliff:g id="high">%1$s</xliff:g></string>
+  <string name="a11y_low_temp">Low: <xliff:g id="low">%1$s</xliff:g></string>
+
+  <!-- Weather Conditions -->
+  <string name="storm">Storm</string>
+  <string name="drizzle">Drizzle</string>
+  <string name="light_rain">Light Rain</string>
+  <string name="moderate_rain">Moderate Rain</string>
+  <string name="heavy_rain">Heavy Rain</string>
+  <string name="intense_rain">Intense Rain</string>
+  <string name="extreme_rain">Extreme Rain</string>
+  <string name="freezing_rain">Freezing Rain</string>
+  <string name="light_shower">Light Shower</string>
+  <string name="shower">Shower</string>
+  <string name="heavy_shower">Heavy Shower</string>
+  <string name="ragged_shower">Ragged Shower</string>
+  <string name="light_snow">Light Snow</string>
+  <string name="snow">Snow</string>
+  <string name="heavy_snow">Heavy Snow</string>
+  <string name="sleet">Sleet</string>
+  <string name="shower_sleet">Shower Sleet</string>
+  <string name="rain_and_snow">Rain and Snow</string>
+  <string name="shower_snow">Shower Snow</string>
+  <string name="mist">Mist</string>
+  <string name="smoke">Smoke</string>
+  <string name="haze">Haze</string>
+  <string name="sand_dust">Sand, Dust</string>
+  <string name="fog">Fog</string>
+  <string name="sand">Sand</string>
+  <string name="dust">Dust</string>
+  <string name="volcanic_ash">Volcanic Ash</string>
+  <string name="squalls">Squalls</string>
+  <string name="clear">Clear</string>
+  <string name="mostly_clear">Mostly Clear</string>
+  <string name="scattered_clouds">Scattered Clouds</string>
+  <string name="broken_clouds">Broken Clouds</string>
+  <string name="overcast_clouds">Overcast Clouds</string>
+  <string name="tornado">Tornado</string>
+  <string name="tropical_storm">Tropical Storm</string>
+  <string name="hurricane">Hurricane</string>
+  <string name="cold">Cold</string>
+  <string name="hot">Hot</string>
+  <string name="windy">Windy</string>
+  <string name="hail">Hail</string>
+  <string name="calm">Calm</string>
+  <string name="light_breeze">Light Breeze</string>
+  <string name="gentle_breeze">Gentle Breeze</string>
+  <string name="breeze">Breeze</string>
+  <string name="fresh_breeze">Fresh Breeze</string>
+  <string name="strong_breeze">Strong Breeze</string>
```

Comment:
I would recommend using case sensitive names so that you know exactly what it holds i.e.

`<string name="Strong_Breeze">Strong Breeze</string>`